math/gunits/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
 
 name             gunits
-version          2.18
+version          2.19
 revision         1
```

Comment:
You need to reset `revision` to `0` when you do an update.

Reply:
Thanks for the reminder; I removed the revision.